src/dynamics/video_player/player/player.js

```diff
@@ -407,6 +407,7 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
                         "tracktagssupport": false,
                         "playbackcount": 0,
                         "playbackended": 0,
+                        "start_content_on_pause": false,
```

Comment:
Also, since this parameter is only used internally and not a configuration option I'm not sure if we should be listing it here.

@jimee02 we could create a separate scope for attributes that are only used internally.

Reply:
Hi both, actually we can leave it as it's, and definitely not include to the `types:{...}` list. Or we have `states` attribute where we can apply `start_content_on_pause` as well.

Reply:
I'd like for there to be a way for engineers to know if a property is injected from attributes or created within the local scope.  Any suggestions for standards here? It could be as simple as a commented section.

Reply:
@jimee02 @sambua @Josewd we could do something like this:
```js
attrs: function() {
    const externalAttrs = {
        // ...
    };
    const internalAttrs = {
        // ...
    };
    return {
        ...externalAttrs,
        ...internalAttrs,
    };
}
```
What do you think?

Reply:
Hi @ntshcalleia , very good idea, but actually we can separate internal and external with `_` prefix. But it could be more relevant to name as stateAttributes, and (mutable or setting)Attributes.

Reply:
the `_` would be confusing to me because i'm expecting this scope to be shared with its children